network-api/networkapi/wagtailpages/pagemodels/blog/blog_topic.py

```diff
@@ -49,28 +49,16 @@ class BlogPageTopic(TranslatableMixin, models.Model):
 
     @staticmethod
     def get_topics():
```

Comment:
Have you considered adding a test for this method?

Reply:
Also, what do think about turning this into a `classmethod`? To me this looks like one, since it does not depend on the instance but does need the class itself.

Reply:
Hi Tibor, this has been updated to use a `classmethod` now!